### PR TITLE
feat(httpapi): Adding stubs for an http api implementation.This com…

### PR DESCRIPTION
**Pull request by jmacelroy**
…mit starts an implementation of an http api
for creating a job to put on a queue for reserving a 
recorder and obtaining an access JWT to use for recording
purposes.

The http api is implemented in typescript and is currently
setup to have separate npm commands from the existing 
implementation.

### CODE REVIEW

**Comment by jmacelroy** [comment]
bah, I did this on a new machine and need to rewrite the commit msg.

File: .eslintignore [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# don't ever lint node_modules
```
Comment (hristoterezov) [inline .eslintignore:1]: Why you reintroduce eslint since we have it in https://github.com/jitsi/jibri-queue/pull/5/files ?

**Review by hristoterezov — COMMENTED** [review]
Why do you commit src/server.ts since it is 0 bytes?

File: .eslintignore (hunk 1 above)
Comment (jmacelroy) [reply .eslintignore:1]: This is config specific to typescript and is branched of master which doesn't have eslint yet. They will eventually get merged. We probably won't be merging the linked pull request into master as is and will most likely pull from it what we need in subsequent PRs.

**Comment by jmacelroy** [comment]
> bah, I did this on a new machine and need to rewrite the commit msg.

Fixed

> Why do you commit src/server.ts since it is 0 bytes?

It's a left over from dev that should be added. I've removed it.

**Pull request closed without merging**